docs/navigation-advanced.md

```diff
@@ -0,0 +1,404 @@
+# Advanced App navigation
+This documentation describes the steps to modify the NavigationService to contain different frames and different navigation levels. You can check out the sample code [here](https://github.com/Microsoft/WindowsTemplateStudio/tree/dev/poc/AdvancedNavigationPaneProject).
```

Comment:
Documentation should not rely on sample. Poc was created to set up the document, but should not be mentioned.